Week06/src/TestClass.java

```diff
@@ -0,0 +1,22 @@
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+
+public class TestClass {
+
+    @Test
+    public void addToInventory(){
+        Animal sausage = new Animal("Sausage", 13, 20, 3, 50, "2019.12.25", -20);
+        Animal newSausage = new Animal("Sausage", 13, 20, 3, 20, "2019.12.25", -20);
+        int result = sausage.quantity + newSausage.quantity;
+        assertEquals(70, result);
```

Comment:
you're not testing anything here... you're storing some values in some variables, then you are doing a calculation and you are checking if your calculation is right... you are not testing the add to inventory